client/cli/src/arg_enums.rs

```diff
@@ -54,17 +54,13 @@ pub const DEFAULT_WASMTIME_INSTANTIATION_STRATEGY: WasmtimeInstantiationStrategy
 #[derive(Debug, Clone, Copy, ValueEnum)]
 #[value(rename_all = "kebab-case")]
 pub enum WasmExecutionMethod {
-	/// Uses an interpreter.
-	#[clap(name = "interpreted-i-know-what-i-do")]
-	Interpreted,
```

Comment:
This should stay and we should instead print some warning that this doesn't exist anymore.